src/commands/deploy/buildImageInAzure/BuildImageStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep } from "@microsoft/vscode-azext-utils";
+import { acrDomain } from "../../../constants";
+import { IBuildImageInAzureContext } from "./IBuildImageInAzureContext";
+import { buildImageInAzure } from "./buildImageInAzure";
+
+export class BuildImageStep extends AzureWizardExecuteStep<IBuildImageInAzureContext> {
+    public priority: number = 225;
+
+    public async execute(context: IBuildImageInAzureContext): Promise<void> {
+        context.registryDomain = acrDomain;
+
+        const run = await buildImageInAzure(context);
+        const outputImages = run?.outputImages;
+        context.telemetry.properties.outputImages = outputImages?.length?.toString();
+
+        if (outputImages) {
```

Comment:
I think we should throw an error here if there's not any output image. The whole create scenario will break at the end anyway, so at least we could give them a more descriptive error?